src/Illuminate/Contracts/Filesystem/Filesystem.php

```diff
@@ -41,8 +39,6 @@ public function get($path);
      *
      * @param  string  $path
      * @return resource|null The path resource or null on failure.
-     *
-     * @throws \Illuminate\Contracts\Filesystem\FileNotFoundException
```

Comment:
Similar here. Perhaps throw an exception instead of null for the other kind of failure too?

Reply:
I've explained this in the main comment of the PR.